libs/application/template-api-modules/src/lib/modules/templates/operating-license/operatingLicense.service.ts

```diff
@@ -76,16 +76,16 @@ export class OperatingLicenseService extends BaseTemplateApiService {
         applicantSsn,
       )
       if (hasCriminalRecord) {
-        throw new TemplateApiError(
-          {
-            title: coreErrorMessages.dataCollectionCriminalRecordTitle,
-            summary: coreErrorMessages.dataCollectionCriminalRecordErrorTitle,
-          },
-          400,
-        )
+        return { success: true }
       }
 
-      return { success: true }
+      throw new TemplateApiError(
+        {
+          title: coreErrorMessages.dataCollectionCriminalRecordTitle,
+          summary: coreErrorMessages.errorDataProvider,
+        },
+        400,
+      )
```

Comment:
The error is swallowed anyway in the duplicate code below
```suggestion
      throw new Error()
```